### PR TITLE
Code from trail project

### PR DESCRIPTION
**Pull request by HectorTorresE**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (HectorTorresE) [inline index.html:35]: Please don't leave unused code as a comment since it makes your code messy and hard to follow

File: index.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
```
Comment (HectorTorresE) [inline index.html:14]: It is recommended to use the  ```section```  tag instead of the ```div```, this will make your code cleaner and easier to understand you can read more about the topic at [HTML Semantic Elements](https://www.w3schools.com/html/html5_semantic_elements.asp)

**Review by HectorTorresE — COMMENTED** [review]
Please check the comments below also please add a readme file with the protect description this will help people identify the reason for this project and what the objective was

File: index.html (hunk 2 above)
Comment (Hamzaoutdoors) [reply index.html:14]: - [x] Please make sure your comment is more polite. So it should also be a suggestion instead of making it look like an order.  For example, you can say, "**It will be better** to use the `section` tag instead of the `div`, this will make your code cleaner and easier to understand, Kindly, work on refactoring your code".  ^_^

- [x] . What do you think to include some links to make your review as an opportunity for the student to learn -->  [HTML Semantic Elements](https://www.w3schools.com/html/html5_semantic_elements.asp) 🦾

File: index.html (hunk 1 above)
Comment (Hamzaoutdoors) [reply index.html:35]: Good job addressing that 👍🏼 👍🏼

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hi @HectorTorresE, I am **Hamza** your reviewer for this PR 😸 

Good job so far 👏🏼 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
( - **N.A.** )

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @HectorTorresE , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
